src/web/arr/trove/plot.arr

```diff
@@ -33,20 +32,27 @@ provide-types {
 import global as G
 import base as B
 import image-structs as I
+import image as IM
 include lists
 import plot-lib as P
 import either as E
 import string-dict as SD
 
 OFFSET = 1
-MAX-SAMPLES = 100000
+MAX_SAMPLES = 100000
```

Comment:
My bad! I thought that's the convention given at https://cs.brown.edu/courses/cs019/2016/Pyret_Style_Guide.html, but I got it wrong. Will change back.